packages/client/hmi-client/src/api/api.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
```

Comment:
If this is the way forward, can probably remove the `uncloak` file if not used for anything

Reply:
There are a few parallel threads along the same veins, let's get the foundations in first, before cleaning up other spots, otherwise this PR is just going to drag on and on.